mmdet3d/apis/inferencers/lidar_seg3d_inferencer.py

```diff
@@ -0,0 +1,199 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import os.path as osp
+from typing import Dict, List, Optional, Sequence, Union
+
+import mmengine
+import numpy as np
+from mmengine.dataset import Compose
+from mmengine.infer.infer import ModelType
+from mmengine.structures import InstanceData
+
+from mmdet3d.registry import INFERENCERS
+from mmdet3d.utils import ConfigType
+from .base_det3d_inferencer import BaseDet3DInferencer
+
+InstanceList = List[InstanceData]
+InputType = Union[str, np.ndarray]
+InputsType = Union[InputType, Sequence[InputType]]
+PredType = Union[InstanceData, InstanceList]
+ImgType = Union[np.ndarray, Sequence[np.ndarray]]
+ResType = Union[Dict, List[Dict], InstanceData, List[InstanceData]]
+
+
+@INFERENCERS.register_module(name='seg3d-lidar')
+@INFERENCERS.register_module()
+class LidarSeg3DInferencer(BaseDet3DInferencer):
```

Comment:
It's a little bit strange to inherit from `BaseDet3DInferencer`, should we add a `BaseSeg3DInferencer`?

Reply:
Probably we will support `VisionSeg3DInferencer` in the future